toolkit/offchain/src/d_param.rs

```diff
@@ -108,12 +108,12 @@ where
 	Ok(McTxHash(res.transaction.id))
 }
 
-fn get_mint_ex_units(response: Vec<OgmiosEvaluateTransactionResponse>) -> anyhow::Result<ExUnits> {
+fn get_mint_ex_units(validators_budgets: Vec<OgmiosEvaluateTransactionResponse>) -> anyhow::Result<ExUnits> {
```

Comment:
Could this and `get_spend_ex_units` be the same function instead? eg `get_ex_units`

Reply:
Yes, I will add a function to `csl` so it could be used in 4 places (maybe more).